bindings/dotnetcore/dotnet-core-binding/Microsoft.Azure.Devices.Gateway/Gateway.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Devices.Gateway
+{
+    /// <summary>
+    ///  Wrapper for Azure IoT Gateway SDK. 
+    /// </summary>
+    public class Gateway
```

Comment:
Two things:
1. If you call it GatewayInterop, then we have this: `Microsoft.Azure.Devices.Gateway.GatewayInterop.Gateway_CreateFromJson` (3x repetition)
2. BrokerInterop isn't part of the public interface.

I'm not saying I love the name `Interop`, but it's better than `Gateway` or `GatewayInterop`.